scripts/globals/wsquest.lua

```diff
@@ -0,0 +1,634 @@
+require('scripts/globals/keyitems');
+require('scripts/globals/quests');
+require('scripts/globals/status');
+
+dsp = dsp or {};
+dsp.wsquest =
+{
+    --------------------------------------
+    -- Quest: The Walls of Your Mind
+    --   NPC: Oggbi (Port Bastok)
+    --    WS: Asuran Fists (Hand-to-Hand)
+    --------------------------------------
+    ['theWallsOfYourMind'] =
+    {
+        ['questName'] = 'The Walls Of Your Mind',
```

Comment:
Is there even any reason to have this? Not used anywhere, and the comments above each block explain which combat skill and weapon it's for.